.env.test

```diff
@@ -13,3 +13,4 @@ DOING=fakeDOING
 DONE=fakeDONE
 MONGODB=mongodb://fakemongo.com
 MONGODBDATABASE=testdatabase
+LOGIN_DISABLED=True
```

Comment:
You could update the README to mention the 3 new environment variables and what they're for.